public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -100,17 +127,23 @@
             TO_CLIENT_ANALYSIS =
             new Function<AchillesAnalysis, org.pmiops.workbench.model.Analysis>() {
                 @Override
-                public org.pmiops.workbench.model.Analysis apply(org.pmiops.workbench.cdr.model.AchillesAnalysis cdr) {
+                public org.pmiops.workbench.model.Analysis apply(AchillesAnalysis cdr) {
+                    List<org.pmiops.workbench.model.AchillesResult> results = new ArrayList<>();
+                    if (!cdr.getResults().isEmpty()) {
+                        results = cdr.getResults().stream().map(TO_CLIENT_ACHILLES_RESULT).collect(Collectors.toList());
```

Comment:
Dumb question: what's the point of having AchillesAnalysis with AchillesResult underneath in the results... how does all this get used in the UI? Do you have a screenshot?

I know I mentioned this before, but Achilles really isn't intended for use in user interfaces / external APIs... continuing to use it that way in our API may make life more difficult for you than if you tailored the API to the UI you were building.

Reply:
So Achilles analysis and results are decent structure for graphing generically on the front end. Flexible and concise ... This survey results api is just how i want it more or less. 
 We have various analyses -- count, gender count, and age decile count. For each concept or question, we have want that array of the analyses with the results for each .  The analyses object is important because it defines what the results mean, display name for the graph, what type of chart to make, what the names of the stratum are.  I could wrap a answer specific class around it. But  the graphing on the front end I'm trying to make as generic as possible -- <app-chart [analysis]=countAnalysis > </chart>  is the interface and it just graphs it.  For basic data we want to graph, it's good.  

We made some new analyses definition just for questions and that helped -- putting the survey_concept_id, question_concept_id, answer_concept_id, answer text, gender/age decile in the stratum. Works great imo.